tests/Offer/OfferTest.php

```diff
@@ -508,6 +509,62 @@ function (Item $item) use ($labels): void {
             ]);
     }
 
+    /**
+     * @test
+     */
+    public function it_handles_replace_labels(): void
+    {
+        $itemId = '9538e4b6-2b8c-404c-93dc-e0dccf8eb175';
+
+        $labels = new Labels(
+            new Label(
+                new LabelName('new_label_1'),
+                true
+            ),
+            new Label(
+                new LabelName('existing_label_1_added_via_ui_and_also_in_new_replace'),
```

Comment:
What does the suffix `_and_also_in_new_replace` mean?

Reply:
It means it is both added by the ui (`LabelAdded`) and also included in the `LabelsReplace` command.
I changed it to **existing_label_1_added_via_ui_and_also_added_in_new_replace_command**